pkg/autoscaler/statforwarder/forwarder.go

```diff
@@ -370,7 +365,12 @@ func (f *Forwarder) maybeRetry(logger *zap.SugaredLogger, s stat, rev string) {
 	f.retryWg.Add(1)
 	go func() {
 		defer f.retryWg.Done()
-		time.Sleep(retryProcessingInterval)
+		// TODO(yanweiguo): Use RateLimitingInterface and NewItemFastSlowRateLimiter.
+		interval := fastRetryProcessingInterval
+		if s.retry > fastMaxProcessRetry {
+			interval = slowRetryProcessingInterval
+		}
+		time.Sleep(interval)
```

Comment:
I am also interested in scaling of this. Probably having a threadpool with ratelimiting queue is going to be the best approach here. I really do not like the explicit sleeps in general code 😄